### DISCUSSION TITLE
Support need

### DISCUSSION THREAD

**Discussion by sillymoon** [quiple/galmuri#48] [General]
Hi there, I'm trying to make this font (Hangul part) support the font i've remade in recent based on *Mojangles*, and I'll inform the creator when i finish.

When metion to my plan to remake the font base on Mojangles, here's the example.
![Pixoles Example](https://github.com/quiple/galmuri/assets/17272494/9f265ad7-9175-4d76-bc8e-24381c86c671)

**Comment by quiple** [comment]
How can I support you specifically?
